server/monitor-types/tailscale-ping.js

```diff
@@ -0,0 +1,89 @@
+// TODO: Handle Case 5
+// In cases where offline peers are not pingable, how should he handle timeout messages? Do we break out of the "ping (ip) timed out" loop and let the server re-call the check function? or do we include the "timed out" string in the expected error messages line and throw the error? (along with "no matching peer" and "is local Tailscale IP")
```

Comment:
I don't like `TODO` or `FIXME` comments as they likely wont be removed.
=> please choose one implementation and add a comment why you chose this way if complex enough.